Microsoft.DotNet.Interactive.PowerShell/DollarProfileHelper.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Interactive.PowerShell
+{
+    using System.Management.Automation;
+
+    internal static class DollarProfileHelper
+    {
+        private const string _profileName = "Microsoft.dotnet-interactive_profile.ps1";
+
+        internal readonly static string AllUsersCurrentHost = GetFullProfileFilePath(forCurrentUser: false);
+        internal readonly static string CurrentUserCurrentHost = GetFullProfileFilePath(forCurrentUser: true);
```

Comment:
nitpick: the order should be `static readonly` by convention.

Reply:
fixed